modules/media/php/module.class.inc

```diff
@@ -58,4 +41,47 @@ class Module extends \Module
     {
         return "Media";
     }
+
+    /**
+     * {@inheritDoc}
+     *
+     * @param string $type    The type of widgets to get.
+     * @param \User  $user    The user widgets are being retrieved for.
+     * @param array  $options A type dependent list of options to provide
+     *                        to the widget.
+     *
+     * @return \LORIS\GUI\Widget[]
+     */
+    public function getWidgets(string $type, \User $user, array $options) : array
+    {
+        switch($type) {
+        case 'candidate':
+            $factory   = \NDB_Factory::singleton();
+            $baseurl   = $factory->settings()->getBaseURL();
+            $candidate = $options['candidate'];
+            if ($candidate === null) {
+                return [];
+            }
+            $db = $factory->database();
+
+            $media = $db->pselect(
+                "SELECT file_name as Filename, date_uploaded as UploadDate
```

Comment:
```suggestion
                "SELECT file_name as Filename, last_modified as LastModified
```
This was changed in https://github.com/aces/Loris/pull/5256

Reply:
done in latest commit, the jsx had to be modified too.